gobblin-test-harness/src/main/java/gobblin/test/setup/config/TestHarnessLauncher.java

```diff
@@ -25,25 +22,10 @@
 public interface TestHarnessLauncher extends ConfigStepsGenerator {
 
   /**
-   *  This method will parse the config based on the input provided
-   *
-   * @param properties The configuration properties
-   */
-
-  public void configure(Properties properties);
```

Comment:
Are we removing the configure and prepareTest methods? How are you planning to send configuration parameters to the harness?